---
title: MSSQLSERVER_26013 | Документация Майкрософт
ms.custom: ''
ms.date: 03/06/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.suite: ''
ms.technology: supportability
ms.tgt_pltfrm: ''
ms.topic: conceptual
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
caps.latest.revision: 9
author: MashaMSFT
ms.author: mathoma
manager: craigg
ms.openlocfilehash: 3ec804b657be554572b4557c9d26918220f1b2a6
ms.sourcegitcommit: f8ce92a2f935616339965d140e00298b1f8355d7
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 07/03/2018
ms.locfileid: "37422923"
---
# <a name="mssqlserver26013"></a>MSSQLSERVER_26013
    
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|26013|  
|Источник события|MSSQLSERVER|  
|Компонент|SQLEngine|  
|Символическое имя|SNI_SSL_READY|  
|Текст сообщения|Сертификат [Cert Hash(sha1) "%hs"] был успешно загружен для шифрования.|  
  
## <a name="explanation"></a>Объяснение  
 Это информационное сообщение. Указанный сертификат был успешно загружен в [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] и может использоваться для шифрования.  
  
## <a name="user-action"></a>Действие пользователя  
 Никаких действий не требуется.  
  
  
